Jamulus.pro

```diff
@@ -95,20 +95,28 @@ win32 {
     contains(CONFIG, "jackonwindows") {
         message(Using Jack instead of ASIO.)
 
-        !exists("C:/Program Files (x86)/Jack/includes/jack/jack.h") {
-            message(Warning: jack.h was not found at the usual place, maybe jack is not installed)
-        }
+        contains(QT_ARCH, "x86") {
+        	ProgramFilesDir = read_registry(HKLM, ProgramFilesDir[, 32])
```

Comment:
As @henkdegroot mentioned, `read_registry` seems to be a [fairly recent Qt addition](https://doc.qt.io/qt-5/qmake-function-reference.html#read-registry-tree-key-flag). Adding this function call will likely break things for pre-5.12.1 Qt users.

Is this really necessary? It's a trade-off between compatibility with earlier Qt and covering even more cases. The existing code uses hardcoded paths as well and nobody seems to have complained, so I'd rather suggest to keep it simple and avoid introducing downsides by doing this?

Reply:
Ok.
On Windows, the user can use the latest QT développement environnement to build and I think it's better to say : Hey, please use the latest environnement.
We are talking about 1% of the user on Windows who will build there own Jamulus app...
I will hard code but don't like that.

Reply:
Hard coded in the last commit:
5aa0168